modules/op-release-notes-1-17.adoc

```diff
@@ -0,0 +1,187 @@
++
+.Example of monitoring the running pipeline run at the namespace level
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  pipeline:
+    # ...
+    metrics.running-pipelinerun.level: namespace
+# ...
+----
+
+[id="operator-new-features-1-17_{context}"]
+=== Operator
+
+* With this update, you can run the `skopeo-copy` command with additional arguments, for example, `--all` and `--preserve-digests`. You can pass these arguments as a space-separated string, as shown in the following example:
++
+.Example command
+[source,terminal]
+----
+$ tkn task start skopeo-copy -p ARGS="--all --preserve-digests"
+----
+
+[id="pac-new-features-1-17_{context}"]
+=== {pac}
+
+* With this update, the `pipelines_as_code_running_pipelineruns_count` metric is added to show the number of running `PipelineRun` resources in {pac}. The metric can be filtered by repository or by namespace.
+
+* With this update, the `pipelines_as_code_pipelinerun_duration_seconds_sum` metric is added to show the total duration in seconds taken by all `PipelineRun` resources in {pac}. The metric can be filtered by repository, namespace, `PipelineRun` status, and `PipelineRun` status change reason.
+
+[id="tekton-chains-new-features-1-17_{context}"]
+=== {tekton-chains}
+
+* With this update, you can configure the `TektonConfig` custom resource (CR) to generate the `x509` key pair of `ecdsa` type and use it with {tekton-chains} to sign artifacts. You can generate the key pair by setting the the `generateSigningSecret` field in the `TektonConfig` custom resource (CR) to `true`:
++
+.Example of creating `ecdsa` key pair
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+# ...
+  chain:
+    disabled: false
+    generateSigningSecret: true 
+# ...
+----
+
+* Before this update, if you did not configure {tekton-chains} in the `TektonConfig` CR, the Operator would not pass any default Chains configuration.
+With this update, if you do not configure {tekton-chains} in the `TektonConfig` CR, the Operator sets the {tekton-chains} configuration with these default properties:
++
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+# ...
+  chain:
+    artifacts.taskrun.format: in-toto
+    artifacts.taskrun.storage: oci
+    artifacts.oci.storage: oci
+    artifacts.oci.format: simplesigning
+    artifacts.pipelinerun.format: in-toto
+    artifacts.pipelinerun.storage: oci
+# ...
+----
+
+* With this update, {tekton-chains} now supports extracting the `mongo-server-url` URL from a specified file that can have any name. You can now use the `storage.docdb.mongo-server-url-path` parameter pointing to a valid file path within the container.
+
+[id="breaking-changes-1-17_{context}"]
+== Breaking changes
+
```

Comment:
As an alternative, you can use the cluster revolver to access tasks that {pipelines-shortname} installs in the `openshift-pipelines` namespace.